### DISCUSSION TITLE
Changelogs

### DISCUSSION THREAD

**Discussion by jinsley8** [resend/react-email#1000] [General]
Can you please release proper changelogs?

It will help tremendously, the current changelog is outdated from April 2023.

Thank you

**Comment by dzienisz** [comment]
Hey. I need that too! from at least some 1.X.X version

**Comment by jinsley8** [comment]
Changelogs have been much better recently. 

They can be viewed in the [releases](https://github.com/resend/react-email/releases) and [docs](https://react.email/docs/changelog)
